From b2d9220ef68d38b8455639a6bb4e47b129608081 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mikeioannina@cyanogenmod.org>
Date: Wed, 13 Jan 2016 18:44:32 +0200
Subject: [PATCH 114/206] Snap: Cleanup focus time duration entries

* Add 3s which was the default before we added the option
* Remove 15/20/30s
* Fix default duration

Change-Id: Ieaad13f248ec9966eefb546beeb6b386156e3b11
---
 res/values/cm_arrays.xml  | 8 ++------
 res/values/cm_strings.xml | 6 ++----
 2 files changed, 4 insertions(+), 10 deletions(-)

diff --git a/res/values/cm_arrays.xml b/res/values/cm_arrays.xml
index 1a44ad17c..126fa7b96 100644
--- a/res/values/cm_arrays.xml
+++ b/res/values/cm_arrays.xml
@@ -28,22 +28,18 @@
     <!-- Touch focus duration -->
     <string-array name="pref_camera_focustime_duration_entries" translatable="false">
         <item>@string/pref_camera_focustime_entry_0</item>
+        <item>@string/pref_camera_focustime_entry_3</item>
         <item>@string/pref_camera_focustime_entry_5</item>
         <item>@string/pref_camera_focustime_entry_10</item>
-        <item>@string/pref_camera_focustime_entry_15</item>
-        <item>@string/pref_camera_focustime_entry_20</item>
-        <item>@string/pref_camera_focustime_entry_30</item>
         <item>@string/pref_camera_focustime_entry_infinite</item>
     </string-array>
 
     <!-- Stored in milliseconds -->
     <string-array name="pref_camera_focustime_duration_values" translatable="false">
         <item>200</item>
+        <item>3000</item>
         <item>5000</item>
         <item>10000</item>
-        <item>15000</item>
-        <item>20000</item>
-        <item>30000</item>
         <item>0x7FFFFFFF</item>
     </string-array>
 
diff --git a/res/values/cm_strings.xml b/res/values/cm_strings.xml
index da19da3e2..5b8ddf306 100644
--- a/res/values/cm_strings.xml
+++ b/res/values/cm_strings.xml
@@ -110,14 +110,12 @@
 
     <!-- Touch focus duration -->
     <string name="pref_camera_focustime_title">Touch focus duration</string>
-    <string name="pref_camera_focustime_default" translatable="false">5000</string>
+    <string name="pref_camera_focustime_default" translatable="false">3000</string>
 
     <string name="pref_camera_focustime_entry_0">0s</string>
+    <string name="pref_camera_focustime_entry_3">3s</string>
     <string name="pref_camera_focustime_entry_5">5s</string>
     <string name="pref_camera_focustime_entry_10">10s</string>
-    <string name="pref_camera_focustime_entry_15">15s</string>
-    <string name="pref_camera_focustime_entry_20">20s</string>
-    <string name="pref_camera_focustime_entry_30">30s</string>
     <string name="pref_camera_focustime_entry_infinite">Infinite</string>
 
     <!-- Shutter speed -->
-- 
2.17.1

